lib/bugsnag/integrations/mongo.rb

```diff
@@ -48,13 +49,31 @@ def leave_mongo_breadcrumb(event_name, event)
         :command_name => event.command_name,
         :database_name => event.database_name,
         :operation_id => event.operation_id,
-        :request_id => event.request_id
+        :request_id => event.request_id,
+        :duration => event.duration
       }
-      meta_data[:duration] = event.duration if defined?(event.duration)
+      if command = pop_command(event.request_id)
+        meta_data[:collection] = command[event.command_name]
```

Comment:
For `getMore` commands, the collection name is in `command['collection']` (and not `command['getMore']`)

Reply:
From looking through their test specifications [here](https://github.com/mongodb/mongo-ruby-driver/tree/master/spec/spec_tests/data/command_monitoring), it looks like `getMore` is the only one.